Assets/Scripts/Data/DataModels/Missions/MissionAccessors.cs

```diff
@@ -7,6 +7,17 @@
     public string Customer { get => customer; set => customer = value; }
     public string Cargo { get => cargo; set => cargo = value; }
     public string Description { get => description; set => description = value; }
+    public MissionUnlockCondition UnlockCondition { get => unlockCondition; }
+    public bool CanBeUnlockedWithMoney
+    {
+         get => UnlockCondition == MissionUnlockCondition.TotalMoney 
```

Comment:
We check the condition type here as well just so this bool isn't used somewhere that doesn't do a check for it. Maybe we unlock one by mistake if it's supposed to be conversation-based and the player has money over the threshold. 

But at the moment we do check in the switch/singleton's unlock method as well so it appears to be redundant for now.